### PR TITLE
Update release notes for 21.6.1

### PR DESCRIPTION
**Pull request by grmoon**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+
+- **Artifact Registry Support**: Now automatically authenticates artifact registry repository urls when uploading container images.
+
+### Updates
+- **Simplified creation of  Kubernetes sample application**: Removed the image repository prompt that was previously displayed to users in the New Project wizard when creating a Kubernetes sample application.
+
+### Bug Fixes
+- Fixed a bug where prompts to install the appropriate language plugin for a selected sample app weren’t displayed when creating a new sample-based project (e.g. the Flask Kubernetes Hello World sample now prompts you to install the python plugin).
+
 ## Version 21.5.1 - Current Release
```
Comment (etanshaul) [inline CHANGELOG.md:25]: remove "Current Release" (why don't we just get rid of that entirely so its impossible to make this error).

Comment (etanshaul) [reply CHANGELOG.md:25]: it's clear enough imo that the release at the top is the latest one

Comment (grmoon) [reply CHANGELOG.md]: agreed

**Review by etanshaul — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+  ![Profile Creation Dialog](docs/images/release-notes/create-profile-dialog.png)
+
+- **Skaffold profile filtering**: Users can now filter for Skaffold profiles with a particular build environment. If you have profiles using multiple build environments, you can easily find all the profiles that use a particular build environment. This feature is particularly useful if you want to build your images locally during development but in-cluster or via Google Cloud Build for production.
+
+  ![Profile Filter](docs/images/release-notes/profile-filter.png)
+
+- **Artifact Registry Support**: Now automatically authenticates artifact registry repository urls when uploading container images.
+
+### Updates
+- **Simplified creation of  Kubernetes sample application**: Removed the image repository prompt that was previously displayed to users in the New Project wizard when creating a Kubernetes sample application.
```
Comment (peterlin741) [inline CHANGELOG.md:20]:
```suggestion
- **Simplified creation of Kubernetes sample application**: Removed the image repository prompt that was previously displayed to users in the New Project wizard when creating a Kubernetes sample application.
```

nit: remove extra space

**Review by peterlin741 — APPROVED** [review]

**Pull request merged**